public/stylesheets/site/2.0/29-role-ie8_or_lower.css

```diff
@@ -1,8 +1,8 @@
 /* IE_CONDITION: IE8_or_lower
 Put any IE-specific overrides into this file -- it will be loaded last and conditionally, only if the browser is IE 8 or below. 
 
-http://otwcode.github.com/docs/front_end_coding/debugging-ie
```

Comment:
IE-related documentation has been removed because we do not support IE and absolutely no one needs to even consider spending time debugging IE in the year 2025.